tests/atest/rules/lengths/empty_test_setup/expected_output.txt

```diff
@@ -1 +1 @@
-test.robot:3:11 [E] 0520 Test Setup does not have any keywords
\ No newline at end of file
+test.robot:3:1 [E] 0520 Test Setup does not have any keywords
```

Comment:
So the old one marked the whole Test Setup, and new one just the beginning? ie:

```
*** Settings  ***
Test Setup    Keyword
```

Old one marked ``Test Setup`` and new one just the beginning of the line.

Reply:
It still underlines the whole `Test Setup`:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/7412964/226831562-4da3c970-bab3-4115-a51a-864ffbe4f713.png">

See: https://github.com/MarketSquare/robotframework-robocop/blob/adjust-rule-message-location/robocop/checkers/lengths.py#L583-L583